### ISSUE TITLE
Failed to create geo URL for 'maps.google.com?q=<place>'

### ISSUE THREAD

**Issue by sleepy-nols** [jakubvalenta/geoshare#14]
**Bug description:**
when playing around with some url's i came across some url's that dont work for me, for example:
```
https://maps.app.goo.gl/v4MDUi9mCrh3mNjz8?g_st=isi
https://maps.app.goo.gl/SghTA2C2z9ix3HAC6?g_st=isi
```
Even when stripping the `?g_st=isi`, no luck. I guess these kind of short link are created in some special share dialogue, which makes them different. will dig a bit deeper on where they come from.

I also found this resolved isse #10 which may be related. as far as I understand these are different short urls though. When entering the special url in a browser this still resolves to an valid url which is again parseable by geoshare, not the short form though.

**Expected behavior:**
- Share special google maps short link with geo share
- The link is converted to geo url and share dialogue pops up. 

**Actual behavior:**
- Share special google maps short link with geo share
- Error pops up: `Failed to create geo URL`

**Comment by sleepy-nols** [comment]
as far as you explain in #12 you basically follow the redirects to resolve the short url to a parseable url.
for example:
```
curl -I "https://maps.app.goo.gl/v4MDUi9mCrh3mNjz8" | grep location:

location: https://maps.google.com?q=Caf%C3%A9+Heinemann,+Bismarckstra%C3%9Fe+91,+41061+M%C3%B6nchengladbach&ftid=0x47b8ac99b0a68bdd:0x8024629be3e9996&entry=gps&lucs=,94224825,94227247,94227248,47071704,47069508,94218641,94233073,94203019,47084304,94208458,94208447&g_ep=CAISDTYuMTQxLjAuOTE2NjAYACDXggMqYyw5NDIyNDgyNSw5NDIyNzI0Nyw5NDIyNzI0OCw0NzA3MTcwNCw0NzA2OTUwOCw5NDIxODY0MSw5NDIzMzA3Myw5NDIwMzAxOSw0NzA4NDMwNCw5NDIwODQ1OCw5NDIwODQ0N0ICREU%3D
```
this would probably need some additional path pattern

**Comment by jakubvalenta** [comment]
Thanks for the bug report! I think you're right that an additional path pattern is needed. I will look into it shortly.

**Comment by jakubvalenta** [comment]
I took the liberty of splitting the second URL into a new issue #16, since it seems an another pattern will be needed to fix it.

**Comment by jakubvalenta** [comment]
Fixed in v1.5.0. It should be available in F-Droid in 3 to 6 days.

**Comment by sleepy-nols** [comment]
cool, thanks for the quick fix :)
